requirements.in

```diff
@@ -12,3 +12,5 @@ git+https://github.com/containerbuildsystem/osbs-client@be934592d8c2a90c186bdab6
 cryptography < 3.5
 requests
 koji
+pycairo
```

Comment:
Shouldn't have to be declared explicitly, pip-compile should find it.

Reply:
it did. Updated.

builddeps.py still added it in build requirements we will have to remember to remove it.